docs/docs/authenticate/README.md

```diff
@@ -8,95 +8,29 @@ or download device configuration files. Optionally,
-# to refresh their access_token. This is verified to work for Google, Okta, and
-# Azure SSO and is used to automatically disconnect a user's VPN if they're removed
-# from the OIDC provider. Leave this disabled if your OIDC provider
-# has issues refreshing access tokens as it could unexpectedly interrupt a
-# user's VPN session.
-default['firezone']['authentication']['disable_vpn_on_oidc_error'] = false
-
-default['firezone']['authentication']['oidc'] = {
-  google: {
-    discovery_document_uri: "https://accounts.google.com/.well-known/openid-configuration",
-    client_id: "<GOOGLE_CLIENT_ID>",
-    client_secret: "<GOOGLE_CLIENT_SECRET>",
-    redirect_uri: "https://firezone.example.com/auth/oidc/google/callback/",
-    response_type: "code",
-    scope: "openid email profile",
-    label: "Google"
-  },
-  okta: {
-    discovery_document_uri: "https://<OKTA_DOMAIN>/.well-known/openid-configuration",
-    client_id: "<OKTA_CLIENT_ID>",
-    client_secret: "<OKTA_CLIENT_SECRET>",
-    redirect_uri: "https://firezone.example.com/auth/oidc/okta/callback/",
-    response_type: "code",
-    scope: "openid email profile offline_access",
-    label: "Okta"
-  }
-}
-```
-
-The following config settings are required for the integration:
-
-1. `discovery_document_uri`: The
-[OpenID Connect provider configuration URI](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig)
-which returns a JSON document used to construct subsequent requests to this
-OIDC provider.
-1. `client_id`: The client ID of the application.
-1. `client_secret`: The client secret of the application.
-1. `redirect_uri`: Instructs OIDC provider where to redirect after authentication.
-This should be your Firezone `EXTERNAL_URL + /auth/oidc/<provider_key>/callback/`
-(e.g. `https://firezone.example.com/auth/oidc/google/callback/`).
-1. `response_type`: Set to `code`.
-1. `scope`: [OIDC scopes](https://openid.net/specs/openid-connect-basic-1_0.html#Scopes)
-to obtain from your OIDC provider. This should be set to `openid email profile`
-or `openid email profile offline_access` depending on the provider.
-1. `label`: The button label text that shows up on your Firezone login screen.
+Firezone supports the following authentication methods:
 
-### Pretty URLs
-
-For each OIDC provider a corresponding pretty URL is created for redirecting to
-the configured provider's sign-in URL. For the example OIDC config above, the
-URLs are:
-
-* `https://firezone.example.com/auth/oidc/google`
-* `https://firezone.example.com/auth/oidc/okta`
+1. [Local email/password](../authenticate/local-auth): Enabled by default. A [time-based
+one time password (TOTP)](../authenticate/multi-factor)
+can optionally be required to add an additional authentication factor.
+1. [Single Sign-On (SSO)](#integrate-a-sso-provider): Enables users to sign
+in using their credentials from Okta, Google, Azure AD, or your own custom identity
```

Comment:
I think we should mention up front the SSO protocols (OIDC) we support now. In the future it could be expanded to SAML, Kerberos, Smart card, OAuth, etc.

```suggestion
in using their credentials from Okta, Google, Azure AD, or any service supporting the OpenID Connect (OIDC) protocol.
```